src/LogicApp/LogicApp.Test/ScenarioTests/WorkflowTests.ps1

```diff
@@ -302,8 +280,8 @@ function Test-UpdateLogicApp
 	try
 	{
 		$workflowName = "82D2D842-C312-445C-8A4D-E3EE9542436D"
-		$definitionFilePath = Join-Path $TestOutputRoot "\Resources\TestSimpleWorkflowTriggerDefinition.json"
-		Set-AzLogicApp -ResourceGroupName $resourceGroupName -Name $workflowName -AppServicePlan $planName -DefinitionFilePath $definitionFilePath -Force
```

Comment:
What happens if the user sends in the AppServicePlan parameter? It is still an option in the cmdlet.

Reply:
So I asked another developer who's been here longer. At one point we allowed App Service plans as an option, but we have long since turned off that functionality (as of like 2 years ago). We still allow a customer to specify one and we record it, but from what I can find out, the API doesn't _do_ anything with that value

Reply:
Will do